packages/config/config/devices/0x027a/zen71.json

```diff
@@ -76,21 +76,36 @@
 			]
 		},
 		"3": {
-			"label": "Auto Turn-Off Timer (Minutes)",
-			"description": "How long the switch remains on",
+			"label": "Auto Turn-Off Timer",
+			"description": "Set the number of minutes to automatically turn off switch once it has been turned on (0 to disable).",
```

Comment:
I've asked @blhoward2 for a review, but I can tell you already that he'll ask you to remove the `(0 to disable)` part. That's what we have `"options"` for.

Reply:
Sure no problem!